coral/src/app/features/topics/acl-request/TopicAclRequest.tsx

```diff
@@ -16,9 +16,19 @@ import topicProducerFormSchema, {
 } from "src/app/features/topics/acl-request/form-schemas/topic-acl-request-producer";
 import { getTopicTeam, TopicTeam } from "src/domain/topic";
 import { AclType } from "src/domain/acl";
+import { environment } from "src/app/features/topics/acl-request/form-schemas/topic-acl-request-shared-fields";
 
 const TopicAclRequest = () => {
+  const navigate = useNavigate();
   const { topicName } = useParams();
+  const [searchParams] = useSearchParams();
+
+  // /topic/aivendemotopic/subscribe route requires an env search param to function correctly
+  // So we redirect when it is missing
+  if (topicName !== undefined && searchParams.get("env") === null) {
+    navigate(`/topic/${topicName}/subscriptions`);
+  }
```

Comment:
I think we also need to handle the case if the env is specified but it does not exist. Same handling would do IMO, redirect to subscriptions.

Reply:
[85de18b](https://github.com/aiven/klaw/pull/1283/commits/85de18bb4c797a85c149be828389d55d22cbb52d) should address it

https://github.com/aiven/klaw/assets/20607294/1c7f5433-8846-4cae-ad28-d50f1fda0619

Reply:
But actually this way is simpler haha [5d2df91](https://github.com/aiven/klaw/pull/1283/commits/5d2df918e70372d7fb44580e1195beca9290bf49)